molo/commenting/templates/comments/comment_done.html

```diff
@@ -1,4 +1,3 @@
-{% extends "base.html" %}
```

Comment:
This shouldn't be removed if it was already there. You just needed to remove the ones that were added while we were working on this